config/app/routes/backoffice/index.yaml

```diff
@@ -10,3 +10,15 @@ backoffice_dashboard_index:
     path: /dashboard
     controller: App\Controller\Backoffice\Dashboard\Index
     methods: [GET]
+
```

Comment:
This can be in a separate file, `routes/backoffice/bookmarks.yaml`.

Reply:
Done - https://github.com/netgen/media-site/pull/185/commits/4f496a3cc63f249e1dd55995cb571f91432cd0be